docs/source/models/concept_models/vivarium_nutrition_optimization/kids/concept_model.rst

```diff
@@ -469,9 +469,40 @@ Requested test runs:
 Subnational Approach
 ~~~~~~~~~~~~~~~~~~~~
 
-.. todo::
-
-  Add general information here on the subnational approach planned. Include: data on which location IDs to use and the percent of the population in each ID. Information on intializing sims with a country and subnational location. What data to pull subnationally vs nationally.
+In order to include SQ-LNS targeting by location, we are switching to 
+use a subnational approach for most data in Wave 3. However, rather 
+than model all subnational locations separately, simulants 
+will just be assigned to a subnational location within their primary 
+location, and have input data pulled for the subnational location instead. 
+Unless otherwise specified in the model request table below, the data outputs 
+do not need to stratified by subnational location.
+
+
+Initializing Simulant's Locations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Simulants will be obtained from the pregnancy sim, the same as in prior 
+waves. These simulants will already have a country location. The pregnancy 
+simulation is only run at the national level.
+
+When these simulants are loaded into the child simulation, they will be 
+assigned a subnational location within their country. Here is the data 
+for the `percent of simulants assigned to each subnational location by sex <https://github.com/ihmeuw/vivarium_research_nutrition_optimization/blob/ff08145109e1434669f08afe702ffc5e3d45a6c2/data_prep/sqlns_subnational/subnational_percents.csv>`_. Note that all 3 countries are included 
+in this csv file. 
+
+Data Inputs: 
+^^^^^^^^^^^^
+
+Once a simulant is assigned to a subnational location, all GBD data used 
+will be subnational specific data. Artifacts will be made for all subnational 
```

Comment:
"all GBD data used"... I was thinking that it might be hard to have subnational specific data for LBWSG (because it would depend on data from the pregnancy sim so we would need to add subnational locations to the pregnancy sim as well) so we were considering only using it for >6 months once simulants are eligible for SQLNS. Also LBWSG PAF calculations are a pain and we could avoid recalculating at the subnational level too.

I think it's fine if we want to take this on, but I think we'll need to add some of these considerations into the strategy if that's the approach we'd like to take!

Reply:
Leaving until we are aligned on the age groups, but I think this will change to clarify that LBWSG comes from pregnancy sim and will be national along with LBWSG PAFs.